cloudferrylib/os/compute/nova_compute.py

```diff
@@ -542,7 +542,7 @@ def change_status(self, status, instance=None, instance_id=None):
             'resume': lambda instance: instance.resume(),
             'paused': lambda instance: instance.pause(),
             'unpaused': lambda instance: instance.unpause(),
-            'suspend': lambda instance: instance.suspend(),
+            'suspended': lambda instance: instance.suspend(),
```

Comment:
I guess here we use not statuses, but actions. And from this side it seems that it should be "suspend", "pause", "unpause" etc.